src/select/typeahead/header/TypeaheadSelectHeader.tsx

```diff
@@ -0,0 +1,43 @@
+import "./_typeahead-select-header.scss";
+
+import React, {Fragment} from "react";
+import classNames from "classnames";
+
+import ListItem from "../../../list/item/ListItem";
+import Tag, {TagShape} from "../../../tag/Tag";
+
+export interface TypeaheadSelectHeaderProps {
+  tags: TagShape[];
+  handleTagRemove: (tag: TagShape) => void;
+  customClassName?: string;
+  input?: React.ReactNode;
+}
+
+function TypeaheadSelectHeader({
+  tags,
+  customClassName,
+  handleTagRemove,
+  input
+}: TypeaheadSelectHeaderProps) {
+  const listClassName = classNames("typeahead-select__header-list", customClassName);
```

Comment:
```suggestion
  const listClassName = classNames("typeahead-select-header__tag-list", customClassName);
```
Since it is a list of tags, I think this makes more sense.

Also, `TypeaheadSelectHeader` is a separate component, so `typeahead-select-header` looks more suitable than `typeahead-select__header`

Reply:
I'm not sure about `tag-list`,  since this list also contains our Input component.